README.md

```diff
@@ -33,7 +72,7 @@ https://k6.io/docs/using-k6/metrics/
 * http_req_sending - Time spent sending data to the remote host
 * http_req_receiving - Time spent receiving response data from the remote host
 
-### Supported countries, transport modes
+### Supported countries, transport modes (deprecated, only applies to time-filter-proto)
```

Comment:
why transport modes are deprecated?

Reply:
because they are not correct, at least for the data I'm using. for example, instead of uk, it should be gb, etc.

Reply:
create different lists for proto and json